recipes/aimrecords/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "aimrecords" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9b562fa5b5109b4b3dd4f83be0061cadbac63fa8031f281b8b5c8ae29967072f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - base58
+  run:
+    - python >=3.6
+    - pytest
+    - ipdb
+    - base58
```

Looking at upstream `pytest` and `ipdb` aren't necessary but `base58` needs to be pinned very strictly: https://github.com/aimhubio/aimrecords/blob/9a6d1cfe3877de633d227afb221e185eb9fc7c4a/setup.py#L30
```suggestion
    - base58 2.0.1
```

---

@xhochy I was able to the pins + additional requirements